File: Java/_12790.java
package Baekjoon;

import java.io.BufferedReader;
import java.io.InputStreamReader;

public class _12790 {

    public static void main(String[] args){

        try {

            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
            int num = Integer.parseInt(br.readLine().trim());

            for(int i = 0; i < num; i++){
                String[] ability = br.readLine().trim().split(" ");
                int hp = max(Integer.parseInt(ability[0]) + Integer.parseInt(ability[4]), 1);
                int mp = max(Integer.parseInt(ability[1]) + Integer.parseInt(ability[5]), 1);
                int power = max(Integer.parseInt(ability[2]) + Integer.parseInt(ability[6]), 0);
                int armor = Integer.parseInt(ability[3]) + Integer.parseInt(ability[7]);
                System.out.println(hp + 5 * mp + 2 * power + 2 * armor);
            }

        } catch (Exception e) { e.printStackTrace(); }
    }

    public static int max(int v1, int v2){
        if(v1 > v2) return v1;
        else return v2;
    }
}

File: Python/12790.py
import sys

f = lambda : sys.stdin.readline().rstrip()

num = int(f())

for i in range(num):
    ability = list(map(int, f().split()))
    hp, mp, power, armor = ability[0], ability[1], ability[2], ability[3]
    hp += ability[4]
    mp += ability[5]
    power += ability[6]
    armor += ability[7]
    if hp < 1:
        hp = 1
    if mp < 1:
        mp = 1
    if power < 0:
        power = 0
    print(hp + 5 * mp + 2 * power + 2 * armor)
